internal/db/schema/migrations/postgres/dev/70_server_tags_migrations.down.sql

```diff
@@ -0,0 +1,8 @@
+begin;
+
```

Comment:
this "down" doesn't really rollback its coresponding "up".

Reply:
Is it supposed to? I thought it was just to destroy things.

Reply:
I guess generally speaking I'm not sure what the point of the "down" bits are. Is it for when you want to make DDL changes that aren't supported in transactions?

Reply:
Leaving alone for now per discussion in Slack.